### PR TITLE
docs: add stings.repeat

### PR DESCRIPTION
**Pull request by rawkode**
Live on cloudnative.tv with @tsandall

### CODE REVIEW

**Comment by tsandall** [comment]
https://deploy-preview-3674--openpolicyagent.netlify.app/

File: docs/content/policy-reference.md [hunk 1]
```diff
@@ -347,6 +347,7 @@ complex types.
 | <span class="opa-keep-it-together">``output := format_int(number, base)``</span> | ``output`` is string representation of ``number`` in the given ``base`` | ✅ |
 | <span class="opa-keep-it-together">``output := indexof(string, search)``</span> | ``output`` is the index inside ``string`` where ``search`` first occurs, or -1 if ``search`` does not exist | ✅ |
 | <span class="opa-keep-it-together">``output := lower(string)``</span> | ``output`` is ``string`` after converting to lower case | ✅ |
+| <span class="opa-keep-it-together">``output := strings.repeat(string, times)``</span> | ``output`` is a ``string`` representing ``string`` repeated N ``times`` | SDK dependant |
```
Comment (ashutosh-narkar) [inline docs/content/policy-reference.md:350]: Nit: What about `output  is ``string`` repeated ``count`` number of times.` ?

Also `SDK dependant` -> `SDK-dependent`

File: topdown/strings.go [hunk 2]
```diff
@@ -423,6 +441,7 @@ func init() {
 	RegisterFunctionalBuiltin1(ast.Upper.Name, builtinUpper)
 	RegisterFunctionalBuiltin1(ast.Lower.Name, builtinLower)
 	RegisterFunctionalBuiltin2(ast.Split.Name, builtinSplit)
+	RegisterFunctionalBuiltin2(ast.Repeat.Name, builtinRepeat)
```
Comment (ashutosh-narkar) [inline topdown/strings.go:444]: `RegisterFunctionalBuiltin2` is deprecated. We should use [RegisterBuiltinFunc](https://github.com/open-policy-agent/opa/blob/main/topdown/builtins.go#L81) instead. Then update the `builtinRepeat` to use [BuiltinFunc](https://github.com/open-policy-agent/opa/blob/main/topdown/builtins.go#L58).

**Review by ashutosh-narkar — COMMENTED** [review]
Thanks for the contribution ! I had couple of comments and also adding some unit tests will be helpful.

**Comment by tsandall** [comment]
This was done as part of the live stream. Closing for now. If we want to come back to this, we can reopen.

**Pull request closed without merging**